src/com/sun/ts/tests/servlet/spec/errorpage/SecondServletErrorPage.java

```diff
@@ -60,7 +61,8 @@ public void service(HttpServletRequest req, HttpServletResponse res)
     pw.println("Servlet Name: " + req.getAttribute(SERVLET_NAME));
     pw.println("Request URI: " + req.getAttribute(REQUEST_URI));
     pw.println("Status Code: " + req.getAttribute(STATUS_CODE));
-    pw.println("Exception Type: " + req.getAttribute(EXCEPTION_TYPE));
+    // with the cast we even enforce it's a Class type
```

Comment:
So, your adding an additional test by casting to Class, to enforce that type Class is returned for STATUS_CODE.  

URLClient.java contains  @assertion_ids for which Servlet spec sections are tested.  Please add an assertion for this additional requirement that your now causing to be enforced.  For example, Servlet:SPEC:104.3.2 is specified in internal/docs/servlet/ServletSpecAssertions.html: 
```The request attribute 'javax.servlet.error.exception_type' of type 'java.lang.Class' will be set if the error page was invoked due to an uncaught exception.
```
internal/docs/servlet:/ServletSpecAssertions.xml:
```
 <assertion required="true" impl-spec="false" defined-by="technology" status="active" testable="true">
                            <id>Servlet:SPEC:104.3.2</id>
                            <description>The request attribute 'javax.servlet.error.exception_type' of type 'java.lang.Class' will be set if the error page was invoked due to an uncaught exception.</description>
                            <location chapter="10" section="9.1"/>
                        </assertion>
```
We also should update the 'javax.servlet' to instead be 'jakarta.servlet' in internal/docs.

Do you agree that the assertion that I pasted already covers the additional test that you are adding with this change?

@markt-asf @stuartwdouglas do you approve this change to enforce that Servlet containers request attribute 'jakarta.servlet.error.exception_type' will be of type java.lang.Class.

Reply:
I agree the assertion you pasted already covers this.

The attribute is already defined as Class in the specification document. See section 10.9.1, table 10-1 (servlet 4.0) so I have no objection to the TCK enforcing this.

Overall the PR looks good to me.